doc/user-guide/How-to-build.md

```diff
@@ -1,71 +1,116 @@
 # How to build MongooseIM
 
+Instructions provided in this page are verified for:
+
+* CentOS 7
+* Ubuntu 16.04 LTS (Xenial)
+* Ubuntu 18.04 LTS (Bionic)
+* macOS 10.14 (Mojave)
+
+For remaining OS versions the instructions should work as well, however some steps or file paths may be different.
```

Comment:
I'd go for `For other OS versions...`. "Remaining" makes it sound like the list of not-stated OS versions is short and has been clarified before (like, we've given a list of OSs before and we're now enumerating just a selection of them).